stable/admin/files/nuoadmin

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+. ${NUODB_HOME}/etc/nuodb_setup.sh
+
+if [ -e ${NUODB_CFGDIR}/jvm-options -a "$NUODB_JAVA_OPTS" = "" ] ; then
+    . ${NUODB_CFGDIR}/jvm-options
+fi
+
+: ${NUODB_SERVERID:=$(hostname)}
+: ${NUODB_ALT_ADDRESS:=$(hostname --fqdn)}
+: ${NUODB_ADMIN_PORT:="48005"}
+
+if [ "${NUODB_ALTERNATIVE_ENTRYPOINT}" != "" ] ; then
+    if nuodocker --api-server "${NUODB_ALTERNATIVE_ENTRYPOINT}:8888" check servers ; then
+        # Admin server exists at alternative entrypoint. Use it instead
+        NUODB_DOMAIN_ENTRYPOINT="${NUODB_ALTERNATIVE_ENTRYPOINT}"
+    fi
+fi
+
+# if endpoint is not specified, default to bootstrap server
+if [ "${NUODB_DOMAIN_ENTRYPOINT}" = "" ] ; then
+    NUODB_DOMAIN_ENTRYPOINT="${NUODB_BOOTSTRAP_SERVERID}"
+fi
```

Comment:
all of this should be handled by the Admin. It seems like you are reinventing the wheel.

Reply:
i'm only mounting the existing nuoadmin file like we do nuosm, nuote etc to allow the additions to it... OK or not?

Reply:
This was one very confusing part of the Admin training session. We should sit down with Adrian and decide how entry points should work. This is what the `peer` property in Admin is for. The training says that you can set it to the load balancer.

Reply:
Hmm, this comment seems unrelated to the code... but like I say this is _the_ nuoadmin file taken from the image, not changed regarding entrypoint.  I added the part that take a copy of the crash directory if it was populated, to avoid overwriting (and when persistent logging allowed it to be there after the crash)...

Reply:
oh. I missed that. Resolving. We can take the discussion offline.